src/styles/styles.js

```diff
@@ -1657,7 +1657,7 @@ const styles = {
         right: -25,
         borderWidth: 3,
         borderRadius: 18,
-        borderColor: themeColors.componentBG,
+        borderColor: themeColors.cardBG,
```

Comment:
How is this change related to this PR?

Reply:
We want to avoid adding keys to the theme file in most situations. But in this case, I agree!